perllib/FixMyStreet/Cobrand/Hackney.pm

```diff
@@ -52,4 +53,61 @@ sub default_map_zoom { 5 }
 
 sub admin_user_domain { 'hackney.gov.uk' }
 
+sub lookup_site_code_config {
+    my ($self, $type) = @_;
+    # uncoverable subroutine
+    # # uncoverable statement
+    {
+        buffer => 1000, # metres
+        url => "https://tilma.mysociety.org/mapserver/hackney", # XXX
+        srsname => "urn:ogc:def:crs:EPSG::27700",
+        typename => $type,
+        property => 'area',
+        accept_feature => sub { 1 },
+    }
+}
+
+sub get_body_sender {
+    my ( $self, $body, $problem ) = @_;
+
+    my $contact = $body->contacts->search( { category => $problem->category } )->first;
+
+    my $parts = join '\s*', qw(^ park : (.*?) ; estate : (.*?) ; other : (.*?) $);
+    my $regex = qr/$parts/i;
+    if (my ($park, $estate, $other) = $contact->email =~ $regex) {
+        my $to = $other;
+        if ($self->lookup_site_code($problem, 'park')) {
+            $to = $park;
+        } elsif ($self->lookup_site_code($problem, 'estate')) {
+            $to = $estate;
+        }
+        $problem->set_extra_metadata(sent_to => { $contact->email => $to });
```

Comment:
Using a hash for `sent_to` is causing a crash when viewing the report at `/report/<id>`. Haven't got to the bottom of why just yet I'm afraid.